cves/kernel/CVE-2012-2745.yml

```diff
@@ -13,32 +13,27 @@ yaml_instructions: |
   not required, but appreciated.
 curated_instructions: |
   If you are manually editing this file, then you are "curating" it.
-
   Set the version number that you were given in your instructions.
-
   This will enable additional editorial checks on this file to make sure you
   fill everything out properly. If you are a student, we cannot accept your work
   as finished unless curated is properly updated.
-curation_level: 0
+curation_level: 2
 reported_instructions: |
   What date was the vulnerability reported to the security team? Look at the
   security bulletins and bug reports. It is not necessarily the same day that
   the CVE was created.  Leave blank if no date is given.
-
   Please enter your date in YYYY-MM-DD format.
 reported_date:
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date.
-
   This is not the same as published date in the NVD - that is below.
-
   Please enter your date in YYYY-MM-DD format.
-announced_date: '2012-08-09'
+announced_date: 'v'
```

Comment:
I'm a litttle confused as to what 'v' means here.. does this mean it's the same as the published date? Or were you just unable to find an announced date? Just looking for clarification.

Reply:
This was a mistake, I will make sure to fix this!